### PR TITLE
Kolyada_EHW1_T1

### PR DESCRIPTION
**Pull request by vklda**

### CODE REVIEW

**Review by vitya771 — APPROVED** [review]
👍🏽

**Review by AnzhelaKhamitskaya — APPROVED** [review]
Lgfm

**Review by MurMkot — APPROVED** [review]

**Review by AlexMackevich — APPROVED** [review]

File: homework/english/kolyada_vlad/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+6  — What does your sister do?
+— She is a programmer.
+
+TASK2
+CORRECT THE GRAMMAR MISTAKES (ИСПРАВЬТЕ ГРАММАТИЧЕСКИЕ ОШИБКИ)
+1 Are you usually work on Saturdays?
+2 Listen! Somebody is singing.
+3 Look! The sun is shining.
+4 He don't use his car every day.
+5 — Where are you come from?
```
Comment (Uchukaknado) [inline homework/english/kolyada_vlad/lecture1/task1.md:18]: Where do you come from?

File: homework/english/kolyada_vlad/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+28 JavaScript variables are containers for storing data values.
+29 Use of Perl/shell style comments ( ) is discouraged.
+30 He's a good programmer and can give you some tips to avoid common coding mistakes. 
+31 I have a big backlog of work to do.
+
+TASK 3
+MAKE YOUR OWN EXAMPLES WITH THE VOCABULARY OF THE SECTION (ПОСТРОЙТЕ СВОИ ПРЕДЛОЖЕНИЯ СО СЛОВАМИ ИЗ СЕКЦИИ). 
+Java is a multithreaded programming language.
+Every week programmers get purpose from product backlog.
+I'll do everything in my power to complete a course.
```
Comment (Uchukaknado) [inline homework/english/kolyada_vlad/lecture1/task1.md:109]: I'll do my best to complete the course.

File: homework/english/kolyada_vlad/lecture1/task1.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+27 Our teacher explained to us the purpose of the variable. 
+28 JavaScript variables are containers for storing data values.
+29 Use of Perl/shell style comments ( ) is discouraged.
+30 He's a good programmer and can give you some tips to avoid common coding mistakes. 
+31 I have a big backlog of work to do.
+
+TASK 3
+MAKE YOUR OWN EXAMPLES WITH THE VOCABULARY OF THE SECTION (ПОСТРОЙТЕ СВОИ ПРЕДЛОЖЕНИЯ СО СЛОВАМИ ИЗ СЕКЦИИ). 
+Java is a multithreaded programming language.
+Every week programmers get purpose from product backlog.
```
Comment (Uchukaknado) [inline homework/english/kolyada_vlad/lecture1/task1.md:108]: Every week programmers get tasks from product backlog.

File: homework/english/kolyada_vlad/lecture1/task1.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+SPEAKING PRACTICE
+ANSWER THE QUESTIONS. INTRODUCE YOURSELF. POST YOUR VIDEO/AUDIO IN YOUR CHAT GROUP. (ОТВЕТЬТЕ НА ВОПРОСЫ. ОПУБЛИКУЙТЕ ВИДЕО.АУДИО ПРЕДСТАВЛЕНИЯ В ЧАТЕ СВОЕЙ ГРУППЫ)
+1 What's your name? 
+My name is Vlad.
+2 How old are you? 
+I am twenty five.
+3 Where do you live?
+I live in Minsk.
+4 What do you do?
+I am train driver. I currently works in subway.
```
Comment (Uchukaknado) [inline homework/english/kolyada_vlad/lecture1/task1.md:122]: I am a train driver. I'm currently working in subway.

**Review by Uchukaknado — COMMENTED** [review]
You rock!

**Pull request merged**